README.md

```diff
@@ -432,6 +432,14 @@ For guidelines on contributing to the project, please refer to our [Contributing
 
 ## Release Notes
 
+### v3.0.0
+
+- Serverless 3 compatibility
+- Test improvlements (splitting unit tests and integration tests)
+  - Tests to run on PRs and integration tests to run on master merge
+- Added workflows for releasing to npm
+- Updated all updatable dependencies
```

Comment:
In my opinion the release notes shouldn't include "chore" type things like tests since they don't affect the end user, and we want to keep this concise

```suggestion
- Added workflows for releasing to npm
- Updated dependencies
```